lib/msf/core/module/platform.rb

```diff
@@ -576,4 +576,14 @@ class Apple_iOS < Msf::Module::Platform
     Alias = "apple_ios"
   end
 
+  #
+  # Unknown
+  # This is a special case for when we're completely unsure of the
+  # platform, such as a crash or default case in code.  Only
+  # utilize this as a catch-all.
+  #
+  class Unknown < Msf::Module::Platform
+    Alias = "unknown"
```

Comment:
What about `Rank`?

Reply:
https://github.com/rapid7/metasploit-framework/blob/cd9ef01c9a09513d6ec923a732ce770214312920/lib/msf/core/module/platform.rb#L12
inherited, no?

Reply:
Yes, but I'd want it to be explicit for this special case.

Reply:
It's also that I don't trust the default value not to change, and an "unknown" platform adds ambiguity to debugging. :/